sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/monitoring/TracingFeignClientIT.java

```diff
@@ -103,7 +122,7 @@ void shouldExtractTraceAndSpanForB3ContextPropagationAndUseInClient() {
         client.exchange(
             RequestEntity.get(
                     URI.create(String.format("http://localhost:%d/api/pingMetrics", port)))
-                .header("x-b3-traceid", "0000000000000001")
+                .header("X-B3-TraceId", "0000000000000001")
```

Comment:
Should it work with any case? Then we maybe need to duplicate the test method.

Reply:
Yes, the correct should be lower case, I fixed it